custom_components/ocpp/api.py

```diff
@@ -867,8 +868,9 @@ def on_boot_notification(self, **kwargs):
             om.charge_point_serial_number.name, None
         )
 
-        asyncio.create_task(self.async_update_device_info(kwargs))
+        self.hass.async_create_task(self.async_update_device_info(kwargs))
         self.hass.async_create_task(self.central.update(self.central.cpid))
+        self.hass.async_create_task(self.post_connect())
```

Comment:
are these tasks executed sequentially or in parallel?

Reply:
In parallel, the Hass create tasks wraps the asyncio function. Figure its best to use Hass one.

Reply:
Realised this could lead to loop, see new commit